README.md

```diff
@@ -7,4 +7,5 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
+Aditya Narayan <narayanaditya95@gmail.com>  
```

Comment:
Two trailing spaces to initiate a line break. Do I use the break tag <br /> instead?